app/controllers/dashboard_controller.rb

```diff
@@ -14,4 +16,22 @@ def index
       end
     end
   end
+
+  def kubectl_config
```

Comment:
I'd try to put as much of this code as possible into a lib (for example in `lib/velum/kubernetes` ?). Then this method should be:

1. Get values from library and set them into instance variables.
2. Render the thing